Style guide updates: placeholders, fake values, and capitalization (#4043)
Kong/docs.konghq.com: app/contributing/style-guide.md

>- Traffic reliability
>- Global observability across all traffic

## Placeholder and example values

The type of placeholder you use depends on context:

* **Generic placeholder values:** In most situations (such as plugin parameters, YAML examples, or Kong configuration), use all caps text and underscores between words.

    For example: `service: SERVICE_NAME`

* **URLs:**
    * For guides with examples that are intended to be runnable as-is, use `localhost` as the domain name.

        For example: `curl -i -X https://localhost:8001/services`

>         If you are following a guide where {{site.base_gateway}} is running on `localhost`, this example can be copy and pasted straight into a terminal.

Reviewer: ```suggestion
        If you are following a guide where {{site.base_gateway}} is running on `localhost`, this example can be copied and pasted straight into a terminal.
```

- Enclose sample code with single backticks.<br/>
  For example: \`sudo yum install /path/to/package.rpm`

## Screenshots

> You can use screenshots to express the capabilities, look and feel, and experience of a feature in situations where exclusively using text would make the documentation harder to understand. We recommend writing the documentation first, **without** using screenshots, and then assessing if a screenshot would enhance the documentation. Screenshots are used to support documentation and not in place of documentation. In some cases using wireframes in place of screenshots are easier to maintain. Otherwise, all screenshots must follow these guidelines. 

Reviewer: ```suggestion
You can use screenshots to express the capabilities, look and feel, and experience of a feature in situations where exclusively using text would make the documentation harder to understand. We recommend writing the documentation first, **without** using screenshots, and then assessing if a screenshot would enhance the documentation. Screenshots are used to support documentation and not in place of documentation. In some cases, using wireframes in place of screenshots are easier to maintain. Otherwise, all screenshots must follow these guidelines. 
```

- Enclose sample code with single backticks.<br/>
  For example: \`sudo yum install /path/to/package.rpm`

## Screenshots

You can use screenshots to express the capabilities, look and feel, and experience of a feature in situations where exclusively using text would make the documentation harder to understand. We recommend writing the documentation first, **without** using screenshots, and then assessing if a screenshot would enhance the documentation. 

Screenshots are used to support documentation and do not _replace_ documentation. In some cases, using wireframes in place of screenshots is easier to maintain. Otherwise, all screenshots must follow these guidelines. 

- Screenshots must be taken with browser developer tools. 
- Resolution should be set to **1500x843.75**.

> - Screenshots of UI elements should include only the relevant **panel**. Panels are container within a UI window which contain multiple related elements.

Reviewer: ```suggestion
- Screenshots of UI elements should include only the relevant **panel**. Panels are a container within a UI window which contain multiple related elements.
```

When deciding which icon to use for a doc, use the following guidelines:

1. Is there a Unicode version?

   We use Unicode for common icons such as ✅ &nbsp; and ❌&nbsp;. You can copy and paste a
   Unicode icon directly into markdown.

2. Is there a Font Awesome icon?

>    We also use a free version of Font Awesome to supplement Unicode icons.

Reviewer 2: ```suggestion
   We also use the free version of Font Awesome to supplement Unicode icons.
```

- Don't use link titles like "Read more" and "Click here". Instead, write descriptive titles that properly detail what content is accessible by clicking the link.
- If the linked content is a larger area like a panel, add a `title` attribute that describes the linked content to the `a` tag.

## Reference style guide

> Follow Kong's style guide whenever possible. However, you can also refer to other external style guides for style and formatting that our guide doesn't cover.

Reviewer 2: ```suggestion
Follow Kong's style guide whenever possible. However, we recommend using the [Google developer style guide](https://developers.google.com/style/) for style and formatting cases that our guide doesn't cover.
```

Reviewer 2: I think it is more confusing to say 'other external style guides' than it is to recommend one explicitly.

> We recommend using the [Google developer style guide](https://developers.google.com/style/).

Reviewer 2: ```suggestion
```
